test/test-debuglet.ts

```diff
@@ -605,59 +628,68 @@ describe('Debuglet', function() {
                {projectId: 'fake-project', credentials: fakeCredentials});
            const debuglet = new Debuglet(debug, defaultConfig);
 
-           const scope =
-               nock(API).post(REGISTER_PATH, function(body: any) {
-                          assert.ok(
-                              _.isUndefined(body.debuggee.labels.minorversion));
-                          return true;
-                        }).once().reply(200, {debuggee: {id: DEBUGGEE_ID}});
+           const scope = nock(API)
+                             .post(
+                                 REGISTER_PATH,
+                                 function(body: any) {
+                                   assert.ok(_.isUndefined(
+                                       body.debuggee.labels.minorversion));
+                                   return true;
+                                 })
+                             .once()
+                             .reply(200, {debuggee: {id: DEBUGGEE_ID}});
 
            // TODO: Determine if the id parameter should be used.
-           debuglet.once('registered', function(_id: string) {
+           debuglet.once('registered', function(id: string) {
              debuglet.stop();
              scope.done();
              done();
            });
            debuglet.start();
          });
 
-      it('should provide minorversion upon registration if on flex', function(
-                                                                         done) {
-        process.env.GAE_SERVICE = 'fake-service';
-        const debug = new Debug(
-            {projectId: 'fake-project', credentials: fakeCredentials});
-        const debuglet = new Debuglet(debug, defaultConfig);
-
-        nocks.oauth2();
-        const scope =
-            nock(API).post(REGISTER_PATH, function(body: any) {
-                       assert.ok(_.isString(body.debuggee.labels.minorversion));
-                       return true;
-                     }).once().reply(200, {debuggee: {id: DEBUGGEE_ID}});
+      it('should provide minorversion upon registration if on flex',
+         function(done) {
+           process.env.GAE_SERVICE = 'fake-service';
+           const debug = new Debug(
+               {projectId: 'fake-project', credentials: fakeCredentials});
+           const debuglet = new Debuglet(debug, defaultConfig);
 
-        // TODO: Determine if the response parameter should be used.
-        debuglet.once('registered', function(_id: string) {
-          debuglet.stop();
-          scope.done();
-          done();
-        });
-        debuglet.start();
-      });
+           nocks.oauth2();
+           const scope = nock(API)
+                             .post(
+                                 REGISTER_PATH,
```

Comment:
Should we move Register_PATH one line up here and elsewhere?

Reply:
`gts fix` made this change.